src/app/pages/virtualization/components/instance-wizard/instance-wizard.component.spec.ts

```diff
@@ -146,17 +153,87 @@ describe.skip('InstanceWizardComponent', () => {
       'Instance Protocol': 'UDP',
     });
 
-    const usbDeviceCheckbox = await loader.getHarness(IxCheckboxHarness.with({ label: 'xHCI Host Controller (0003)' }));
-    await usbDeviceCheckbox.setValue(true);
+    // TODO: https://ixsystems.atlassian.net/browse/NAS-133118
```

Comment:
If tests get broken as part of your changes, it's best to fix them in this PR.
Plus we need new cases for creating a VM.
Please communicate with Rehan on how to do this..